src/mixins/createStatefulChildrenMixin.ts

```diff
@@ -170,7 +170,7 @@ function manageChildren(evt: StateChangeEvent<StatefulChildrenState>): void {
 			}
 			else {
 				/* Tuple of Promise, child ID, position in child list */
-				resolvingWidgets.push([ internalState.registry.get(id), id, key ]);
+				resolvingWidgets.push([ internalState.registry.has(id).then((exists) => { if (exists) { return internalState.registry.get(id); } }), id, key ]);
```

Comment:
Do you intend to do so?  Also, is there anyway to avoid this complexity of promises?  I guess I am wondering if the `registry.get()` behaviour is not right, because the result of this new promise chain is basically *if the registry has the widget, then return the instance, otherwise return undefined*.  Why doesn't `.get()` just behave like that?